circuitClient.js

```diff
@@ -116,19 +119,20 @@ class CircuitClient {
       self.client.addEventListener('basicSearchResults', searchResultHandler);
       self.client.addEventListener('searchStatus', searchStatusHandler);
 
-      searchId = await self.client.startBasicSearch([{
-        scope: Circuit.Enums.SearchScope.CONVERSATIONS,
-        searchTerm: query
-      }]);
+      searchId = await self.client.startBasicSearch([
+        {
+          scope: Circuit.Enums.SearchScope.CONVERSATIONS,
+          searchTerm: query
+        }
+      ]);
     });
   }
 
   /**
    * logon
    */
-  logon (accessToken) {
-    return this.client.logon(accessToken ? {accessToken: accessToken} : undefined)
-      .then(user => console.log(`Logged on to Circuit: ${user.displayName}`))
+  logon(accessToken) {
+    return this.client.logon(accessToken ? { accessToken: accessToken } : undefined).then(user => console.log(`Logged on to Circuit: ${user.displayName}`));
```

Comment:
Not sure you used a style formatter, but our style guidelines are to put the ".then" on a new line

Reply:
Done